packages/clay-management-toolbar/src/SearchForm.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import React from 'react';
+
+interface IProps extends React.FormHTMLAttributes<HTMLFormElement> {
+	onlySearch?: boolean;
+}
+
+const SearchForm: React.FunctionComponent<IProps> = ({
+	children,
+	method = 'get',
+	onlySearch = false,
+	...otherProps
+}) => {
+	const content = (
+		<form {...otherProps} method={method} role="search">
```

Comment:
Would be a good idea ship this code to `ClayForm`?

Reply:
Not sure about that, I still don't see a real need to create a component to render just the form.